app/components/PageHead.tsx

```diff
@@ -36,6 +36,11 @@ export const PageHead = (props: PageHeadProps) => {
       )}
       <meta property="og:type" content="website" />
       <meta property="og:locale" content={locale || "en"} />
+
+      <meta property="twitter:card" content="summary_large_image" />
+      <meta property="twitter:title" content={props.mediaTitle} />
+      <meta property="twitter:description" content={props.metaDescription} />
+      <meta property="twitter:image" content={props.mediaImageSrc} />
```

Comment:
this src is optional. I'm not sure what happens if you render this with an empty `content` string. Probably best to conditionally render it

Reply:
Yeah sure.